src/vulkan/vulkanutils.h

```diff
@@ -4,9 +4,31 @@
 
 #include <iostream>
 #include <string>
+#include <vector>
+
+#if defined _WIN32
+#include <windows.h>
+#endif // _WIN32
 
 namespace dw::vulkan {
 
+// Define the VulkanRTLPtr type based on platform
+#if defined (_WIN32)
+ typedef HMODULE VulkanRTLPtr;
```

Comment:
Why not always cast to `void*` and get rid of these macros?

Reply:
you would still have to handle the `HMODULE` case somewhere though.. right? Let me ponder

Reply:
I was just thinking casting it to a void* instead on windows, would make the code cleaner here

Reply:
But then you can cast it back.. or you'd have to do some reinterpret_casts when using it.. right?

Reply:
Oh okay, didn't see where it was used. I thought the generic LoadFunction could take a void*, but I see that one is windows specific as well. never mind then